firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -496,7 +496,8 @@ public void beginRealtimeHttpStream() {
             String.format(
                 "Unable to connect to the server. Try again in a few minutes. HTTP status code: %d",
                 responseCode);
-        // Return server message for when the Realtime API is disabled and the server returns a 403
+        // Return server message for when the Firebase Remote Config Realtime API is disabled and
```

Comment:
Since this is the name of the API in the Cloud console (and we can't easily change it), I think we should keep it for continuity so it's obvious what we're referring to.

Reply:
Agree it's not ideal

Reply:
Makes sense